src/plots/cartesian/index.js

```diff
@@ -126,6 +126,7 @@ exports.finalizeSubplots = function(layoutIn, layoutOut) {
  */
 exports.plot = function(gd, traces, transitionOpts, makeOnCompleteCallback) {
     var fullLayout = gd._fullLayout;
+    if(!fullLayout._subplots) return;
```

Comment:
What's this about? When do we encounter this?

Reply:
Sometimes happen at the end of `transition_test` with an error similar to the image below:
![undefinedSubplot](https://user-images.githubusercontent.com/33888540/107406091-0a1ab480-6ad6-11eb-9978-9c71c4b8e88d.png)

Reply:
Simply for reference:
https://app.circleci.com/pipelines/github/plotly/plotly.js/2974/workflows/734c1977-517c-43b8-88bf-9fe83e3c5e55/jobs/103632